### PR TITLE
Move instruction structure to vm

### PR DESCRIPTION
**Pull request by Juan-M-V**

### CODE REVIEW

File: src/vm/instruction.rs [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-num Register {
-    AP,
-    FP
-}
-
 pub struct Instruction {
-    off0: BigUint,
-    off1: BigUint,
-    off2: BigUint,
+    off0: i16,
```
Comment (unbalancedparentheses) [inline src/vm/instruction.rs:4]: out of curiosity why are you changing the type @Juan-M-V ?

Comment (Juan-M-V) [reply src/vm/instruction.rs:4]: Because the specification divides the 64-bit words into off0, off1, off2 and flags, each taking 16-bits

Comment (unbalancedparentheses) [reply src/vm/instruction.rs:4]: Cool, thanks

**Comment by unbalancedparentheses** [comment]
Is the PR ready to review? If it is still a draft please update it to be a draft. Thanks @Juan-M-V

**Comment by Juan-M-V** [comment]
@unbalancedparentheses The PR is ready for review!

**Pull request merged**